elements/rh-audio-player/rh-audio-player.css

```diff
@@ -448,337 +443,294 @@ rh-menu > button:focus {
   background-color: var(--_text-color);
 }
 
-#play svg,
-#full-play svg {
-  fill: var(--rh-audio-player-icon-background-color, var(--_surface-color));
-}
-
-#playback-rate-tooltip {
-  margin-inline: 10px 2px;
-}
-
-#playback-rate-stepper,
-#full-playback-rate-stepper {
-  display: flex;
-  align-items: center;
-}
-
-#playback-rate,
-#full-playback-rate,
-#playback-rate-suffix,
-#full-playback-rate-suffix {
-  color: var(--_text-color);
-  font-size: var(--rh-font-size-code-md, 1rem);
-  font-family: var(--rh-font-family-code, RedHatMono, 'Red Hat Mono', 'Courier New', Courier, monospace);
-}
-
-#playback-rate,
-#full-playback-rate {
-  appearance: none;
-  line-height: 36px;
-  padding: 2px 0;
-  border: none;
-  color: var(--_text-color);
-  background-color: var(--_surface-color);
+#play,
+#full-play {
+  color: var(--rh-audio-player-icon-background-color, var(--_surface-color));
```

Comment:
This causes the play button in `rh-audio-player.html` to be invisible (white foreground, white background).